library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java

```diff
@@ -47,7 +43,39 @@ public static boolean isEmoji(String message) {
     }
 
     public static boolean containsMarkdown(String message) {
-        return Pattern.compile(MARKDOWN_PATTERN).matcher(message).find();
+        if (checkSymbolBrackets(message, "*") || checkSymbolBrackets(message, "_")
+                || checkSymbolBrackets(message, "#")
+                || checkSymbolBrackets(message, "**") || checkSymbolBrackets(message, "__")
+                || message.contains("~~") || message.contains("#{")
+                || message.contains("---") || message.contains("***") || message.contains("___")
+                || message.contains(">{")
+                || checkSymbolBrackets(message,"`") || checkSymbolBrackets(message, "```") || message.contains("//")
+                || message.contains("<i>") || message.contains("<em>") || message.contains("<cite>") || message.contains("<dfn>")
+                || message.contains("<b>") || message.contains("<strong>")
+                || message.contains("<sup>") || message.contains("<sub>")
+                || message.contains("<u>") || message.contains("<ins>")
+                || message.contains("<a>") || message.contains("<ul>") || message.contains("<ol>")
+                || message.contains("<img>") || message.contains("<blockquote>")
+                || message.contains("<h1>") || message.contains("<h2>") || message.contains("<h3>")
+                || message.contains("<h4>") || message.contains("<h5>") || message.contains("<h6>")
+                || message.startsWith("# ") || message.startsWith("## ") || message.startsWith("### ")
+                || message.startsWith("#### ") || message.startsWith("##### ")
+        ) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean containsMention(String message) {
+        return message.contains("@");
```

Comment:
@andriizhumela , it looks wrong. in case message is "Test@", it is not mention text.
The function name is weird.

Reply:
I don't think the function needs in here since your containMarkdown method contains it already.

Reply:
@adrian09h no, we need that function (it was renamed to `isContainsMention`) because we do not check for the "@" symbol, we check mentions with `message.getMentionedUsers()`. 
Example in `getDeletedOrMentionedText()` function

Also i removed testcase @MarkDown, because @ it is not correct creteria.

Reply:
![Screenshot_20191204-150851](https://user-images.githubusercontent.com/14316323/70145177-2b00a200-16a8-11ea-89bd-9e571d146d37.png)

Reply:
You updated `containsMention` method correctly. (it was wrong before.)